CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Removed
 - Deprecated `refresh_db` removed from pkgrepo state. Use `refresh` instead. (#57366)
 - Deprecated internal functions salt.utils.locales.sdecode and .sdecode_if_string removed. Use salt.utils.data.decode instead. (#57367)
 - Removed deprecated misc. internal Salt functions. See https://github.com/saltstack/salt/issues/57368 for more info. (#57368)
+- Remove salt/utils/vt.py duplication from filename map. (#57004)
```

Comment:
This isn't a feature removal - also it's only in the `tests` dir, so really not user-facing.